types/muicss/muicss-tests.tsx

```diff
@@ -0,0 +1,200 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {
+    Appbar,
+    Button,
+    Checkbox,
+    Col,
+    Container,
+    Divider,
+    Dropdown,
+    DropdownItem,
+    Form,
+    Input,
+    Option,
+    Panel,
+    Radio,
+    Row,
+    Select,
+    Tab,
+    Tabs,
+    Textarea
+} from 'muicss/react';
```

Comment:
Could you actually point out if these imports will work at runtime? [I don't see an `index.js` that would correspond to this.](https://unpkg.com/muicss@0.9.36/lib/react/)

Reply:
Yes, they work at runtime.

They resolve to the `react.js` file in root, not a directory's `index.js` file.

https://unpkg.com/muicss@0.9.36/

I'm using these types in my application now.